packages/plugin-commands-installation/src/install.ts

```diff
@@ -124,6 +124,10 @@ export function help () {
             description: `The directory in which the ${WANTED_LOCKFILE} of the package will be created. Several projects may share a single lockfile`,
             name: '--lockfile-dir <dir>',
           },
+          {
+            description: 'Allows to install dependencies to a directory other than node_modules',
```

Comment:
More consistent with `lockfile-dir`:

The directory in which dependencies will be installed (instead of node_modules)